x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -268,6 +273,25 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
     }
   };
 
+  // show this when wildcard filename with matches operator
+  const getWildcardWarning = (precedingWarning: string): React.ReactNode => {
+    return (
+      <p>
+        {precedingWarning}{' '}
+        <EuiIconTip
+          type="iInCircle"
+          content={
+            <FormattedMessage
+              id="xpack.lists.exceptions.builder.exceptionMatchesOperator.warningMessage.wildcardInFilepath"
+              defaultMessage="To make a more efficient Event Filter, use multiple conditions and make them as specific as possible when using wildcards in the path values. For instance, adding a process.name or file.name field."
```

Comment:
Should this be `To make a more efficient event filter, use multiple...` without capital letters?

Reply:
@kevinlog @caitlinbetz ☝️

Reply:
yes, thanks @dasansol92.